source/patterns/@aws-solutions-constructs/aws-s3-eventbridge/README.md

```diff
@@ -0,0 +1,106 @@
+# aws-s3-eventbridge module
+<!--BEGIN STABILITY BANNER-->
+
+---
+
+![Stability: Experimental](https://img.shields.io/badge/stability-Experimental-important.svg?style=for-the-badge)
+
+> All classes are under active development and subject to non-backward compatible changes or removal in any
+> future version. These are not subject to the [Semantic Versioning](https://semver.org/) model.
+> This means that while you may use them, you may need to update your source code when upgrading to a newer version of this package.
+
+---
+<!--END STABILITY BANNER-->
+
+| **Reference Documentation**:| <span style="font-weight: normal">https://docs.aws.amazon.com/solutions/latest/constructs/</span>|
+|:-------------|:-------------|
+<div style="height:8px"></div>
+
+| **Language**     | **Package**        |
+|:-------------|-----------------|
+|![Python Logo](https://docs.aws.amazon.com/cdk/api/latest/img/python32.png) Python|`aws_solutions_constructs.aws_s3_eventbridge`|
+|![Typescript Logo](https://docs.aws.amazon.com/cdk/api/latest/img/typescript32.png) Typescript|`@aws-solutions-constructs/aws-s3-eventbridge`|
+|![Java Logo](https://docs.aws.amazon.com/cdk/api/latest/img/java32.png) Java|`software.amazon.awsconstructs.services.s3eventbridge`|
+
+This AWS Solutions Construct implements an Amazon S3 bucket connected to an Amazon EventBridge.
+
+Here is a minimal deployable pattern definition in Typescript:
+
+``` typescript
+import { S3ToEventbridge, S3ToEventbridgeProps } from '@aws-solutions-constructs/aws-s3-eventbridge';
+import { EventbridgeToStepfunctions, EventbridgeToStepfunctionsProps } from '@aws-solutions-constructs/aws-s3-stepfunctions';
+import * as stepfunctions from '@aws-cdk/aws-stepfunctions';
+
+const startState = new stepfunctions.Pass(stack, 'StartState');
+
+const s3ToEventbridge = new S3ToEventbridge(this, 's3-eventbridge', {});
+const eventBus = s3ToEventbridge.eventBus;
+
+new EventbridgeToStepfunctions(this, 'test-s3-stepfunctions-stack', {
+    stateMachineProps: {
+      definition: startState
+    },
+    existingEventBusInterface: eventBus
+});
+```
+
+## Initializer
+
+``` text
+new S3ToEventbridge(scope: Construct, id: string, props: S3ToEventbridgeProps);
+```
+
+_Parameters_
+
+* scope [`Construct`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_core.Construct.html)
+* id `string`
+* props [`S3ToEventbridgeProps`](#pattern-construct-props)
+
+## Pattern Construct Props
+
+| **Name**     | **Type**        | **Description** |
+|:-------------|:----------------|-----------------|
+|existingBucketObj?|[`s3.IBucket`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-s3.IBucket.html)|Existing instance of S3 Bucket object. If this is provided, then also providing bucketProps is an error. |
+|bucketProps?|[`s3.BucketProps`](https://docs.aws.amazon.com/cdk/api/latest/docs/@aws-cdk_aws-s3.BucketProps.html)|Optional user provided props to override the default props for the S3 Bucket.|
```

Comment:
I think this construct "responds to S3 events" - while not directly, it's the same idea, we may want to control the events that come out of S3 that are passed on to EventBridge.